lib/std/os/windows/bits.zig

```diff
@@ -929,9 +928,12 @@ pub usingnamespace switch (builtin.arch) {
             SegSs: DWORD,
             ExtendedRegisters: [512]BYTE,
 
-            pub fn getRegs(ctx: *const CONTEXT) struct {bp: usize, ip: usize} {
-                return .{.bp = ctx.Ebp, .ip = ctx.Eip};
-            }  
+            pub fn getRegs(ctx: *const CONTEXT) struct {
+                bp: usize,
+                ip: usize,
```

Comment:
This change was made by `zig fmt` and it slipped into this PR.
@LemonBoy did #4342 fix this unwanted change?

Reply:
This is another different and unrelated problem `zig fmt` has.

Reply:
Should I make an issue for it?

Reply:
https://github.com/ziglang/zig/issues/4349